execution/client/config.go

```diff
@@ -29,7 +29,7 @@ import (
 const (
 	defaultDialURL                 = "http://localhost:8551"
 	defaultRPCRetries              = 3
-	defaultRPCTimeout              = 900 * time.Millisecond
+	defaultRPCTimeout              = 2 * time.Second
```

Comment:
this is the previous default, see https://github.com/berachain/beacon-kit/pull/2362/files#r1921088464